### PR TITLE
docs: Remove helm installer from doc

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

## What does this pull request change?
Remove helm installer from doc

## What issues does this pull request fix or reference?
https://github.com/eclipse/che/issues/20552

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [x] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by themr0c** [comment]
@tstastna will work on it (but GitHub doesn't let me add her as assignee).

**Comment by tstastna** [comment]
I'll take a look.

File: modules/installation-guide/partials/proc_installing-che-on-kubernetes_using_chectl.adoc [hunk 1]
```diff
@@ -4,83 +4,84 @@
 // installing-{prod-id-short}-on-aws
 // installing-{prod-id-short}-on-microsoft-azure
 
-[id="installing-{prod-id-short}-on-kubernetes-using-{prod-cli}_and_helm_{context}"]
+[id="installing-{prod-id-short}-on-kubernetes-using-{prod-cli}_{context}"]
 = Installing {prod-short} on {k8s-platform} using {prod-cli}
```
Comment (themr0c) [inline modules/installation-guide/partials/proc_installing-che-on-kubernetes_using_chectl.adoc]:
```suggestion
= Installing {prod-short} on {kubernetes} using {prod-cli}
```

**Comment by tstastna** [comment]
@tolusha I have submitted a PR with my suggestions to your branch.

**Review by themr0c — APPROVED** [review]

**Review by tstastna — APPROVED** [review]

**Review by l0rd — APPROVED** [review]

**Pull request merged**